en/controllers/components/csrf.rst

```diff
@@ -68,6 +68,10 @@ token.
     FormHelper. If you do not, you will need to manually create hidden inputs in
     each of your forms.
 
+To manually add the CSRF token to the response object, you can put below code piece inside your form::
+
+    <?= $this->Form->create();?>
```

Comment:
Putting a form inside another form won't do what you want. Are you aiming to output just the CSRF token field?

Reply:
But if this is put inside another form it will generate invalid HTML.

Reply:
I just checked the source and you're right, it generates a new from with the CSRF token in it. but it didn't give me any errors.
Is there a way to output just the CSRF token field inside a form?

Reply:
You *could* make one with `$this->Form->hidden('_csrfToken', ['value' => $this->request->getParam('_csrfToken')]);` but there isn't a public method that just emits a CSRF token field.